codes/codes.go

```diff
@@ -25,7 +25,13 @@ import (
 	"strconv"
 )
 
-// A Code is an unsigned 32-bit error code as defined in the gRPC spec.
+// A Code is a status code according to the gRPC spec:
+//
+// https://github.com/grpc/grpc/blob/master/doc/statuscodes.md
```

Comment:
Easwar made me switch urls to a certain format: https://github.com/grpc/grpc-go/blob/master/xds/internal/balancer/wrrlocality/balancer.go#L19 (I think this is a strong heuristic of .go files). Can you please switch this?

Reply:
Done